<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdressDetailInput</class>
 <widget class="QWidget" name="AdressDetailInput">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AdressDetailInput</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>491</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QLineEdit" name="Name">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>姓名</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_6">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>单位</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_3">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>备注</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_4">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>职务</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_7">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>邮箱</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QLineEdit" name="Mobile2">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="Telephone">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_8">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>手机1</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLineEdit" name="Identity">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLineEdit" name="Remark">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="label_9">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>手机2</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_10">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>座机</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="Mobile1">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="Unit">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="Email">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="Upload">
        <property name="font">
         <font>
          <family>等线</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>保存</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Cancel">
        <property name="font">
         <font>
          <family>等线</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>清空</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
